sdk/python/kubeflow/training/constants/constants.py

```diff
@@ -19,52 +19,56 @@
 APISERVER_TIMEOUT = 120
 
 # TFJob K8S constants
-TFJOB_GROUP = 'kubeflow.org'
-TFJOB_KIND = 'TFJob'
-TFJOB_PLURAL = 'tfjobs'
-TFJOB_VERSION = os.environ.get('TFJOB_VERSION', 'v1')
+TFJOB_GROUP = "kubeflow.org"
+TFJOB_KIND = "TFJob"
+TFJOB_PLURAL = "tfjobs"
+TFJOB_VERSION = os.environ.get("TFJOB_VERSION", "v1")
 
-TFJOB_LOGLEVEL = os.environ.get('TFJOB_LOGLEVEL', 'INFO').upper()
+TFJOB_LOGLEVEL = os.environ.get("TFJOB_LOGLEVEL", "INFO").upper()
+
+TFJOB_BASE_IMAGE = "docker.io/tensorflow/tensorflow:2.9.1"
```

Comment:
It might be nice to use the default image with GPU support same as `PYTORCHJOB_BASE_IMAGE`.
WDYT?

Reply:
@tenzen-y I think the problem is Tensorflow GPU image is 2.2Gb more than image with CPU support: https://hub.docker.com/r/tensorflow/tensorflow/tags?page=1&name=2.9.1.
Maybe we can introduce `TFJOB_BASE_IMAGE` and `TFJOB_BASE_IMAGE_GPU` in our SDK, what do you think ?

Reply:
What are your thoughts on that @johnugeorge @tenzen-y @anencore94 ?

Reply:
I also think introducing both cpu,gpu is nice to have, but set the cpu image as the default image would be safe.

Reply:
@tenzen-y I wasn't able to find official PyTorch image with CPU support only: https://hub.docker.com/r/pytorch/pytorch/tags
Are you aware of those ?

Reply:
Sorry, I assumed it existed...

Reply:
@tenzen-y @anencore94 I've added the Tensorflow GPU Image in the constants.

Reply:
Thanks for updating!

Reply:
Thanks a lot !